web/src/rest.js

```diff
@@ -1,14 +1,19 @@
 
+import axios from 'axios';
 import { RestClient } from '@girder/components/src';
 
 const apiRoot = process.env.VUE_APP_API_ROOT || 'http://localhost:8080/api/v1';
+const publishApiRoot = process.env.PUBLISH_APP_API_ROOT || 'http://localhost:8000/api';
```

Comment:
The `VUE_APP_` prefix is magic, and is necessary for the value to be injected: https://cli.vuejs.org/guide/mode-and-env.html#environment-variables

Perhaps `VUE_APP_PUBLISH_API_ROOT`?